app/src/types/user.js

```diff
@@ -2,22 +2,80 @@
 
 "use strict";
 
-export type User = {
-  +created_at: number,
-  +token: string,
-  +spid_idp: string,
-  +fiscal_code: string,
-  +name: string,
-  +familyname: string,
-  +spidcode?: string,
-  +gender?: string,
-  +mobilephone?: string,
-  +email?: string,
-  +address?: string,
-  +expirationdate?: string,
-  +digitaladdress?: string,
-  +countyofbirth?: string,
-  +dateofbirth?: string,
-  +idcard?: string,
-  +placeofbirth?: string
-};
+import * as t from "io-ts";
+
+const UserModel = t.intersection([
```

Comment:
please create a story (tagged as technical debt) for making these types more specific (i.e. fiscalcode must be matched against a pattern, as we do [here](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/api/definitions/FiscalCode.ts))

Reply:
Done: https://www.pivotaltracker.com/story/show/154134211